moor_generator/lib/src/analyzer/options.dart

```diff
@@ -88,6 +88,9 @@ class MoorOptions {
   @JsonKey(name: 'named_parameters', defaultValue: false)
   final bool generateNamedParameters;
 
+  @JsonKey(name: 'named_parameters_always_required', defaultValue: false)
+  final bool namedParametersAlwaysRequired;
```

Comment:
Could you also add a quick description of this parameter to `docs/pages/docs/Advanced Features/builder_options.md`?

Reply:
Done